ecosante/newsletter/models.py

```diff
@@ -327,6 +329,18 @@ def today_episodes(self):
             current_app.logger.error(e)
             return [dict()]
 
+    @property
+    def today_indice_uv(self):
+        if not self.indice_uv:
+            return dict()
+        data = self.indice_uv['data']
+        try:
+            return next(iter([v for v in data if v['date'] == str(self.date)]), dict())
```

Comment:
Je ne connais pas `iter`, ça sert à quoi ?
Aussi la valeur par défaut de next va faire qu’on ne va pas lever `StopIteration`, et qu’on ne logguera pas, je pense que c’est mieux de l’enlever

Reply:
J'imagine que c'est pour créer une itération, j'ai repris la même forme que `today_forecast` mais effectivement, je confirme qu'on filtre déjà en amont sur la date.